browser/mocha/replace.js

```diff
@@ -22,6 +22,10 @@ extendInterfaces('replace', function(context, teardown) {
         // implementation for later:
         var originalInstanceTemplate = Polymer.Base.instanceTemplate;
 
+        if (Polymer.Base.instanceTemplate.isSinonProxy) {
```

Comment:
So I'm not suggesting that the wrapper is replaced, I'm suggesting that we only stub `instanceTemplate` once, with a stub that's actually capable of replacing multiple elements. This way we don't have a stub calling a stub calling a stub, each doing a querySelector and replace. It would have to be slightly rewritten to do a tree walk rather than querySelector, but that should be relatively easy.